bin/gh-pages

```diff
@@ -39,32 +40,57 @@ function main(args) {
       '.'
     )
     .option('-n, --no-push', 'Commit only (with no push)')
+    .option("-l, --localuser", "Use Git local user config (not global)")
+    .option("-u, --user <username,email>", "Override default username, email")
     .parse(args);
 
-  ghpages.publish(
-    path.join(process.cwd(), program.dist),
-    {
-      repo: program.repo,
-      silent: !!program.silent,
-      branch: program.branch,
-      src: program.src,
-      dest: program.dest,
-      message: program.message,
-      tag: program.tag,
-      dotfiles: !!program.dotfiles,
-      add: !!program.add,
-      only: program.remove,
-      remote: program.remote,
-      push: !!program.push
-    },
-    function(err) {
-      if (err) {
-        process.stderr.write(err.message + '\n');
-        return process.exit(1);
+  if (program.localuser) {
+    Promise.all([
+      new Git().exec("config", "--local", "user.name"),
+      new Git().exec("config", "--local", "user.email")
+    ]).then(results => {
+      publish({
+        name: results[0].output.trim(),
+        email: results[1].output.trim()
+      });
+    });
+  } else if (program.user) {
+    var userParts = program.user.split(",");
+    publish({
+      user: userParts[0],
```

Comment:
[Above](https://github.com/tschaub/gh-pages/pull/229/files#diff-2a9efa0b08eba0bbe5896df72f13afd6R53), this key is `name`, not `user`.

Reply:
Cheers, i'll have a look at this in the next few days